client/service_logs.go

```diff
@@ -26,7 +27,7 @@ func (cli *Client) ServiceLogs(ctx context.Context, serviceID string, options ty
 	if options.Since != "" {
 		ts, err := timetypes.GetTimestamp(options.Since, time.Now())
 		if err != nil {
-			return nil, err
+			return nil, errors.Wrap(err, "invalid value for \"since\"")
```

Comment:
nit: 
<pre>
`invalid value for "since"`
</pre>

Reply:
Updated 👍